designs/jane/src/body.mjs

```diff
@@ -0,0 +1,219 @@
+export const body = {
+  name: 'jane.body',
+  measurements: [
+    'chest',
+    'hips',
+    'hpsToWaistBack',
+    'waistToKnee',
+    'shoulderToShoulder',
+    'neck',
+    'hpsToBust',
+    'waistToFloor',
+  ],
+  options: {
+    bodyEase: { pct: 21, min: 21, max: 50, menu: 'fit' },
+    neckDepthFront: { pct: 31, min: 25, max: 50, menu: 'style' },
+    neckDepthBack: { pct: 18, min: 15, max: 50, menu: 'style' },
+    neckWidth: { pct: 71, min: 65, max: 85, menu: 'style' },
+    shiftLength: { pct: 2, min: 0, max: 20, menu: 'style' },
+  },
+
+  draft: function draftJaneBody({
+    options,
+    Point,
+    Path,
+    points,
+    paths,
+    Snippet,
+    snippets,
+    sa,
+    paperless,
+    macro,
+    measurements,
+    part,
+  }) {
+    //body
+    const lengthBody =
+      (measurements.waistToKnee + measurements.hpsToWaistBack) * (1 + options.shiftLength)
+
+    const workingHip =
+      measurements.chest > measurements.hips ? measurements.chest / 2 : measurements.hips / 2
+
+    const widthBody = workingHip * (1 + options.bodyEase)
+
+    const maxLength =
+      lengthBody > measurements.waistToFloor + measurements.hpsToWaistBack
+        ? measurements.waistToFloor + measurements.hpsToWaistBack
+        : lengthBody
+
+    points.topLeft = new Point(0, 0)
+    points.topRight = new Point(widthBody, 0)
+    points.bottomLeft = new Point(0, maxLength )
+    points.bottomRight = new Point(widthBody, maxLength )
+
+    points.gorestartLeft = points.bottomLeft.shiftTowards(points.topLeft, maxLength / 2)
+    points.gorestartRight = points.bottomRight.shiftTowards(points.topRight, maxLength / 2)
+
+    points.middle = points.topLeft.shiftTowards(points.topRight, widthBody / 2)
+    points.leftShoulder = points.middle.shiftTowards(
+      points.topLeft,
+      measurements.shoulderToShoulder / 2
+    )
+    points.rightShoulder = points.middle.shiftTowards(
+      points.topRight,
+      measurements.shoulderToShoulder / 2
+    )
+
+    paths.sideseam = new Path()
```

Comment:
Is it me, or is `sideseam` a bit misleading as name for this path?
Because this seam to be the entire garment outline.